order/views.py

```diff
@@ -88,10 +80,9 @@ def dydx_order(self, request):
             orders = admin.get_account()
             orders = vars(orders)
             orders_data = orders["data"]["account"]["openPositions"]
+            if orders_data is {}:
```

Comment:
`if not orders_data` is a better way I think cause it does the job for both {} and None